unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/LoadingScreen/Scripts/LoadingScreenController.cs

```diff
@@ -148,28 +154,39 @@ private bool IsNewRealm()
         private bool IsNewScene(Vector2Int currentDestinationCandidate) =>
              worldState.GetSceneNumberByCoords(currentDestinationCandidate).Equals(-1);
 
-        private void CheckSceneTimeout(Vector2Int currentDestinationCandidate)
+        private bool IsSceneLoaded(Vector2Int candidate) =>
+            worldState.GetScene(worldState.GetSceneNumberByCoords(candidate))?.loadingProgress >= 100;
+
+        private void IsPlayerLoaded()
         {
-            //If we are settling on the destination position, but loading is not complete, this means that kernel is calling for a timeout.
-            //For now, we hide the loading screen and add a notification
-            if (currentDestinationCandidate.Equals(currentDestination) &&
-                worldState.GetScene(worldState.GetSceneNumberByCoords(currentDestination))?.loadingProgress < 100)
-            {
-                notificationsController.ShowNotification(new Model
-                {
-                    message = "Loading scene timeout",
-                    type = Type.GENERIC,
-                    timer = 10f,
-                    destroyOnFinish = true
-                });
+            //We have to check if the player is loaded
+            if (commonDataStore.isPlayerRendererLoaded.Get())
                 FadeOutView();
+            else
+            {
+                percentageController.SetAvatarLoadingMessage();
+                commonDataStore.isPlayerRendererLoaded.OnChange += PlayerLoaded;
             }
+
         }
 
         private void FadeOutView()
         {
+            timeoutCTS.Cancel();
             view.FadeOut();
             loadingScreenDataStore.decoupledLoadingHUD.visible.Set(false);
         }
+
+        private void DoTimeout()
+        {
+            notificationsController.ShowNotification(new Model
+            {
+                message = "Loading scene timeout",
+                type = Type.GENERIC,
+                timer = 10f,
```

Comment:
Let's make this 60s
```suggestion
                timer = 60f,
```

Reply:
That one is the time the notification remains on, I think you meant to modify LOAD_SCENE_TIMEOUT, which is currently at 120 seconds. Right?

Reply:
Thanks!